docs/en/setup/backend/backend-alarm.md

```diff
@@ -152,7 +165,42 @@ Currently, metrics from the **Service**, **Service Instance**, **Endpoint**, **S
 
 Submit an issue or a pull request if you want to support any other scopes in Alarm.
 
-## Webhook
+## Hooks
+Hooks are a way to send alarm messages to the outside world. SkyWalking supports multiple hooks of the same type, each hook can support different configurations. 
+For example, you can configure two Slack hooks, one named `default` and set `isGlobal: true` means this hook will apply on all `Alarm Rules` **without config** `specific-hooks`.
+Another named `custom1` will only apply on the `Alarm Rules` which **with config** `specific-hooks` and include the name `slackHooks.custom1`.
+
+```yaml
+hooks:
+  slackHooks:
+    default:
+      isGlobal: true # if true, this hook will apply on all rules, unless a rule has its own specific hook.
```

Comment:
Will mutiple global rules conflict? Or we will pick one randomly?

Reply:
support multiple global hooks in one type, if they have different names

Reply:
So, how would we react when conflicts? Are we still forwarding all as users say so?

Reply:
If use the `yaml` settings, we can't config the same name under one item. such as we can't define 2 `default` name under slackHooks

Reply:
Syntactic restriction

Reply:
The `default` is just a name and has no special meaning, users can define it themself.